README.md

```diff
@@ -86,58 +90,48 @@ Expected output:
 
 ```console
 NAME:   csi-secrets-store
+LAST DEPLOYED: Mon Nov  4 12:51:55 2019
```

Comment:
let's avoid dates here as it makes the readme look out of date

Reply:
done!